CHANGELOG.next.asciidoc

```diff
@@ -208,6 +208,10 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Add persistent volume claim name to volume if available {pull}38839[38839]
 - Raw events are now logged to a different file, this prevents potentially sensitive information from leaking into log files {pull}38767[38767]
 
+*Agentbeat*
```

Comment:
This is an interesting case, I would say this belongs in the agent release notes instead of being here but having to do two PRs is a massive pain.

Reply:
Unless this also comes with user facing documentation on how someone would use it, you could probably omit it from the changelog. Especially if this is primarily for internal use.

Reply:
https://github.com/elastic/elastic-agent/pull/4849 created for elastic-agent changelog entry